src/config/mod.rs

```diff
@@ -69,12 +69,12 @@ create_config! {
 
     // Ordering
     reorder_extern_crates: bool, true, false, "Reorder extern crate statements alphabetically";
-    reorder_extern_crates_in_group: bool, true, false, "Reorder extern crate statements in group";
-    reorder_imports: bool, false, false, "Reorder import statements alphabetically";
-    reorder_imports_in_group: bool, false, false, "Reorder import statements in group";
+    reorder_extern_crates_in_group: bool, false, false, "Reorder extern crate statements in group";
```

Comment:
Why is the default value for `reorder_extern_crates_in_group` set to `false`?

Reply:
I feel it is uncommon to group `extern crate`s in the way that it is common to group `use` imports and therefore there is no need for the grouping and instead we should do what is simplest and gives the most consistent formatting.

Reply:
I personally feel that it is not that uncommon to group `extern crate`s in some logical way (e.g. serde + log + rustc_private + local + others).

Also IMHO rerodering in groups gives the most consistent formatting, especially when `#[macro_use]` gets involved (we cannot reorder stuffs across items with `#[macro_use]`).

For example, with grouping disabled,

```rust
#[macro_use]
extern crate serde_derive;
extern crate serde_json;
extern crate serde;

extern crate semver;
extern crate toml;
extern crate itertools;
```

the above file will be formatted like this:

```rust
#[macro_use]
extern crate serde_derive;
extern crate itertools;
extern crate semver;
extern crate serde;
extern crate serde_json;
extern crate toml;
```

but if we put the `serde` group to the bottom,

```rust
extern crate semver;
extern crate toml;
extern crate itertools;

#[macro_use]
extern crate serde_derive;
extern crate serde_json;
extern crate serde;
```

it will be formatted like this:

```rust
extern crate itertools;
extern crate semver;
extern crate toml;
#[macro_use]
extern crate serde_derive;
extern crate serde;
extern crate serde_json;
```